### ISSUE TITLE
The detailed configuration of the reproduced model CENet_1.0x

### ISSUE THREAD

**Issue by liaozhihao2757** [cardwing/Codes-for-PVKD#29]
我下载了您提供的模型，但是模型的训练配置，如batchsize、weight decay等仍是未知，请问您能否提供一下配置文件（arch_cfg.yml）或一些训练细节，非常期待您的回复！
I downloaded the reproduced model CENet_1.0x, but the training configuration of the CENet, such as batchsize and weight decay, is still unknown. Can you provide the configuration file (arch_cfg.yml) or other training details? Looking forward to your reply!

**Comment by cardwing** [comment]
You can just follow the official codes of [CENet](https://github.com/huixiancheng/CENet).
